doc/tut2.rst

```diff
@@ -1083,3 +691,9 @@ JavaScript-compatible code you should remember the following:
 - ``cstring`` in JavaScript means JavaScript string. It is a good practice to
   use ``cstring`` only when it is semantically appropriate. E.g. don't use
   ``cstring`` as a binary data buffer.
+
+
+Part 3
+======
+
+Next part will be entirely about metaprogramming via macros: `Part III <tut3.html>`_
```

Comment:
I don't think many people will reach to the end of Part 2 to read this line because of the sentence in bold at the top - 
`Note that this document is somewhat obsolete as the manual contains many more examples of the advanced language features.` So maybe instead of `Nim Tutorial Part - 3`, make it `Nim Macro Tutorial` and link it directly from https://nim-lang.org/documentation.html?

Reply:
well that is a good point. But I can't find the sources of the website int he Nim repository. So I guess that would be outside of the scope of this PR.

Reply:
Yes, it is in a separate repo - https://github.com/nim-lang/website/blob/master/jekyll/documentation.html